frontend/src/pages/NewRunV2.tsx

```diff
@@ -85,6 +86,8 @@ function NewRunV2(props: NewRunV2Props) {
   const [isStartingNewRun, setIsStartingNewRun] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
   const [isParameterValid, setIsParameterValid] = useState(false);
+  const [isPipelineNameValid, setIsPipelineNameValid] = useState(false);
+  const [pipelineNameErrMsg, setPipelineNameErrMsg] = useState('');
```

Comment:
Do we need `isPipelineNameValid` state here? Can I assume that `pipelineNameErrMsg` is empty means pipeline name is valid?